src/components/HomePage.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export const HomePage:React.FC = () => {
+  return (
```

Comment:
you can remove word `return` and just return jsx in round brackets `() => (`

check all such components